tools/clang/test/CodeGenSPIRV_Lit/attribute.numthreads.missing.hlsl

```diff
@@ -1,4 +1,4 @@
 // RUN: not %dxc -T cs_6_0 -E main -fcgl  %s -spirv  2>&1 | FileCheck %s
 
-// CHECK: 4:6: error: thread group size [numthreads(x,y,z)] is missing from the entry-point function
```

Comment:
Should we be deleting the code that generated this error message?

Reply:
Unclear. It's in the SPIR-V emitter, so it might technically still be reachable given invalid dxil. Do you have any insight here?

Reply:
My read of the code in the SpirvEmitter is that it shouldn't be reachable.

Any thoughts from @cassiebeckley, @sudonatalie, @Keenuts, or @s-perron?

Reply:
Looking at it I'm convinced you're right - I've replaced it with an assert

Reply:
Yes, this looks fine to me.